yarn-project/aztec-cli/src/index.ts

```diff
@@ -96,7 +97,7 @@ export function getProgram(log: LogFn, debugLogger: DebugLogger): Command {
 
   program
     .command('generate-private-key')
-    .description('Generates a 32-byte private key.')
+    .description('Generates a private key which fits into the field used by Grumpkin curve.')
```

Comment:
```suggestion
    .summary('Generates an encryption private key.')
    .description('Generates a private key which fits into the field used by Grumpkin curve, can be used as an encryption private key.')
```

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/1919/commits/324d193c9b96e2edd844ac10f46a2c1d264d9fc1. Didn't use your suggestion directly since I think it's clearer to include there the term scalar field.